single-campaign.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Template Variables for Campaigns.
+ *
+ * @package P4MT
+ */
+
+use Timber\Timber;
+
+// Initializing variables.
+$context = Timber::get_context();
+
+/**
+ * Post object.
+ *
+ * @var P4_Post $post
+ * */
+$post            = Timber::query_post( false, 'P4_Post' );
+$context['post'] = $post;
+
+// Save custom style settings.
+$custom_styles = [];
+
+// Set specific CSS for Montserrat.
+$pf = $post->campaign_header_primary;
+
+$header_font_style = [
+	'Montserrat'       => "font-family: 'Montserrat' !important; font-weight: 900 !important;",
+	'Montserrat_Light' => "font-family: 'Montserrat' !important; font-weight: 500 !important;",
+];
+
+if ( $pf && array_key_exists( $pf, $header_font_style ) ) {
+	$header_font = $header_font_style[ $pf ];
+} else {
+	$header_font = `font-family: {$pf} !important;`;
+}
+
+$custom_styles['css']['nav_color']               = $post->campaign_nav_color ? ".navbar { background-color: {$post->campaign_nav_color} !important;}" : null;
+$custom_styles['nav_type']                       = $post->campaign_nav_type;
+$custom_styles['campaign_logo_color']            = $post->campaign_logo_color ?? 'light';
+$custom_styles['css']['header_color']            = $post->campaign_header_color ? " h1, h2, h3, h4, h5 { color: {$post->campaign_header_color} !important;}" : null;
+$custom_styles['css']['campaign_header_primary'] = $post->campaign_header_primary ? " h1, h2, h3, h4, h5 { {$header_font} }" : null;
+$custom_styles['css']['header_serif']            = $post->campaign_header_serif ? " .page-header { font-family: {$post->campaign_header_serif}!important;}" : null;
+$custom_styles['css']['header_sans']             = $post->campaign_header_sans ? " .page-header { font-family: {$post->campaign_header_sans} !important;}" : null;
+$custom_styles['css']['body_font']               = $post->campaign_body_font ? " body, p { font-family: '{$post->campaign_body_font}' !important;}" : null;
+$custom_styles['css']['btn_primary']             = $post->campaign_primary_color ? " .btn-primary { background: {$post->campaign_primary_color} !important; border-color: {$post->campaign_primary_color} !important;}" : null;
+$custom_styles['css']['btn_secondary']           = $post->campaign_secondary_color ? " .btn-secondary:hover { background: $post->campaign_secondary_color !important; border-color: {$post->campaign_primary_color}; }" : null;
+$custom_styles['css']['anchor']                  = $post->campaign_secondary_color ? " a { color: {$post->campaign_secondary_color } !important; }" : null;
```

Comment:
not a big issue, but the rules here override wp admin's rules

` a { color: #25784f !important; }		`

![image](https://user-images.githubusercontent.com/15197444/55710577-939d5f00-59f3-11e9-8f1d-4ef7fb379b06.png)